app/src/androidTest/java/ch/epfl/culturequest/ProfileCreatorActivityTest.java

```diff
@@ -166,5 +160,9 @@ public void wrongUsernamesFailProfileCreation() {
     }
 
 
+    @AfterClass
+    public static void destroy() {
+        Intents.release();
+    }
```

Comment:
I don't think this is useful since the Intents are already released in the release method.

Reply:
yep, removed it